modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseR2DBCDatabaseContainer.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers.containers;
```

Comment:
Make sure it is using `ClickHouseContainer` from `org.testcontainers.clickhouse`

```suggestion
package org.testcontainers.clickhouse;
```

Reply:
@livk-cloud check this comment.